pkg/apis/triggers/v1beta1/event_listener_validation.go

```diff
@@ -232,9 +232,6 @@ func containerFieldMask(in *corev1.Container) *corev1.Container {
 	out.Image = ""
 	out.Args = nil
 	out.Ports = nil
-	out.LivenessProbe = nil
-	out.ReadinessProbe = nil
-	out.StartupProbe = nil
```

Comment:
this will break the scenario for customresource

Reply:
Why?

Reply:
Added all changes in this commit https://github.com/tektoncd/triggers/pull/1620/commits/fed390f91ad29bb77e027fb3832f7bdadbeb6ec9